src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+    private Color color;
+
+    public Figure(Color color) {
+        this.color = color;
+    }
+
+    public Color getColor() {
+        return color;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: - , area: - units, sides: - units, color: -");
+    }
+
+    @Override
+    public int obtainArea() {
+        return 0;
+    }
```

Comment:
There is no need to override methods in an abstract class unless you have common logic for each child element
```suggestion
```